aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -76,6 +81,17 @@ func resourceAwsApiGatewayRestApiCreate(d *schema.ResourceData, meta interface{}
 
 	d.SetId(*gateway.Id)
 
+	if body, ok := d.GetOk("body"); ok {
+		_, err := conn.PutRestApi(&apigateway.PutRestApiInput{
+			RestApiId: gateway.Id,
+			Mode:      aws.String("overwrite"),
+			Body:      []byte(body.(string)),
+		})
+		if err != nil {
+			return fmt.Errorf("Error putting API Gateway specification: %s", err)
```

Comment:
Could you use `errwrap.Wrapf("Error creating API Gateway specification: {{err}}", err)` there please? (notice the creating word, to make it clear we are putting by creating)